<ui version="4.0" >
 <class>DictionaryWidget</class>
 <widget class="QWidget" name="DictionaryWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dictionary</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="backwardButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Previous</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="qdictionary.qrc" >:/resources/prev.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Next</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="qdictionary.qrc" >:/resources/next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <property name="sizeAdjustPolicy" >
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SearchLine" name="lineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="searchButton" >
       <property name="text" >
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>72</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="filteringCheckBox" >
       <property name="toolTip" >
        <string>Filter Results</string>
       </property>
       <property name="text" >
        <string>Filtering</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="filterWidget" >
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterComboBox" >
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SearchLine" name="filterLineEdit" />
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLine</class>
   <extends>QLineEdit</extends>
   <header>searchline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qdictionary.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>filteringCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>471</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
